src/main/java/uk/gov/hmcts/ccd/data/definition/UIDefinitionRepository.java

```diff
@@ -38,30 +38,30 @@ public SearchResult getSearchResult(final String caseTypeId) {
         return cachedUiDefinitionGateway.getSearchResult(version.getVersion(), caseTypeId);
     }
 
-    public SearchInputDefinition getSearchInputDefinitions(final String caseTypeId) {
+    public SearchInputFieldsDefinition getSearchInputDefinitions(final String caseTypeId) {
```

Comment:
shall we rename the method names too?

Reply:
renamed to getSearchInputFieldDefinitions